airbyte-integrations/connectors/destination-exasol/src/main/java/io/airbyte/integrations/destination/exasol/ExasolDestination.java

```diff
@@ -0,0 +1,95 @@
+import io.airbyte.db.factory.DatabaseDriver;
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.db.jdbc.JdbcUtils;
+import io.airbyte.integrations.base.Destination;
+import io.airbyte.integrations.base.IntegrationRunner;
+import io.airbyte.integrations.base.ssh.SshWrappedDestination;
+import io.airbyte.integrations.destination.NamingConventionTransformer;
+import io.airbyte.integrations.destination.jdbc.AbstractJdbcDestination;
+import io.airbyte.protocol.models.v0.AirbyteConnectionStatus;
+import java.util.HashMap;
+import java.util.Map;
+import javax.sql.DataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ExasolDestination extends AbstractJdbcDestination implements Destination {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ExasolDestination.class);
+
+  public static final String DRIVER_CLASS = DatabaseDriver.EXASOL.getDriverClassName();
+
+  public ExasolDestination() {
+    super(DRIVER_CLASS, new ExasolSQLNameTransformer(), new ExasolSqlOperations());
+  }
+
+  public static void main(String[] args) throws Exception {
+    new IntegrationRunner(new ExasolDestination()).run(args);
+  }
+
+  @Override
+  public AirbyteConnectionStatus check(JsonNode config) {
+    final DataSource dataSource = getDataSource(config);
+    try {
+      final JdbcDatabase database = getDatabase(dataSource);
+      final NamingConventionTransformer namingResolver = getNamingResolver();
+      final String outputSchema = namingResolver.getIdentifier(config.get("schema").asText());
+      attemptTableOperations(outputSchema, database, namingResolver, getSqlOperations(), true);
+      return new AirbyteConnectionStatus().withStatus(AirbyteConnectionStatus.Status.SUCCEEDED);
+    } catch (final Exception e) {
+      LOGGER.error("Exception while checking connection: ", e);
+      return new AirbyteConnectionStatus()
+          .withStatus(AirbyteConnectionStatus.Status.FAILED)
+          .withMessage("Could not connect with provided configuration.\n" + e.getMessage());
+    } finally {
+      try {
+        DataSourceFactory.close(dataSource);
+      } catch (final Exception e) {
+        LOGGER.warn("Unable to close data source.", e);
+      }
+    }
+  }
+
+  @Override
+  public JsonNode toJdbcConfig(final JsonNode config) {
+    final String jdbcUrl = String.format(DatabaseDriver.EXASOL.getUrlFormatString(),
+            config.get(JdbcUtils.HOST_KEY).asText(), config.get(JdbcUtils.PORT_KEY).asInt());
+
+    final ImmutableMap.Builder<Object, Object> configBuilder = ImmutableMap.builder()
+        .put(JdbcUtils.USERNAME_KEY, config.get(JdbcUtils.USERNAME_KEY).asText())
+        .put(JdbcUtils.JDBC_URL_KEY, jdbcUrl);
+
+    if (config.has(JdbcUtils.PASSWORD_KEY)) {
+      configBuilder.put(JdbcUtils.PASSWORD_KEY, config.get(JdbcUtils.PASSWORD_KEY).asText());
+    }
+
+    if (config.has(JdbcUtils.SCHEMA_KEY)) {
+      configBuilder.put("schema", config.get(JdbcUtils.SCHEMA_KEY).asText());
```

Comment:
is there a default schema which can be used if the user doesn't provide one in the configuration?

Reply:
Exasol does not create a default schema at startup. The user has to create one manually.